### PR TITLE
Completed initial test suite for getPhoto in PostgreSQL

### PR DESCRIPTION
**Pull request by kraig-droid**
1. Main change is the new test file to time queries
2. Also simplified the PostgreSQL queries -- I didn't need all the "client.release" statements, so the code is much simpler

### CODE REVIEW

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

File: test/db.index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  .then((minMaxResults) => {
+    minPhotoId = minMaxResults.rows[0].min;
+    maxPhotoId = minMaxResults.rows[0].max;
+
+    describe('Database', function () {
+      after(db.end);
+
+      describe('getPhoto retrieves random and beginning photos',
+        function () {
+          const photoIds = Array.from({ length: numOfTests },
```
Comment (priscilagonzalez) [inline test/db.index.js]: `Array.from` this is new to me. :)

File: test/db.index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/* eslint-disable func-names */
+/* eslint-disable prefer-arrow-callback */
+// ARROW CALLBACKS ARE DISCOURAGED BY MOCHA BECAUSE THEY MESS UP TESTING
```
Comment (priscilagonzalez) [inline test/db.index.js]: Thanks!

Comment (priscilagonzalez) [inline test/db.index.js]: Thanks!

File: test/db.index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+const { expect } = chai;
+
+const db = require('../database');
+
+const numOfTests = 20; // How many photos to retrieve, for instance
+
+let minPhotoId;
+let maxPhotoId;
+
+// Because we want to have these values to construct tests, mocha
```
Comment (priscilagonzalez) [inline test/db.index.js]: Helpful comment :)

**Review by azdoolittle — APPROVED** [review]
lgtm

**Pull request merged**